test/persist-catalog-migration/mzcompose.py

```diff
@@ -401,3 +405,43 @@ def check_objects(c: Composition, i: int):
     """
         )
     )
+
+
+def workflow_test_version_skips(c: Composition) -> None:
```

Comment:
I'm not sure if this is the best file for this, but I didn't know where else to put it.